### PR TITLE
chore: adhere to ruff's `C`

### PR DESCRIPTION
**Pull request by dni**
note: max-complexity should be decreased to 10 in another PR.

### CODE REVIEW

File: lnbits/nodes/lndrest.py [hunk 1]
```diff
@@ -167,6 +167,8 @@ async def close_channel(
         point: Optional[ChannelPoint] = None,
         force: bool = False,
     ):
+        if short_id:
```
Comment (motorina0) [inline lnbits/nodes/lndrest.py:170]: did `ruff` suggested this? how can it know?

Comment (dni) [reply lnbits/nodes/lndrest.py:170]: ups, that was a pyright, it was an unused local variable

File: pyproject.toml [hunk 2]
```diff
@@ -179,4 +180,4 @@ dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
 
 [tool.ruff.lint.mccabe]
 # Unlike Flake8, default to a complexity level of 10.
```
Comment (prusnak) [inline pyproject.toml:182]:
```suggestion
# Unlike Flake8, default to a complexity level of 16.
```

**Comment by prusnak** [comment]
@dni can you have a look at this CI failure? Is it related to the PR? https://github.com/lnbits/lnbits/actions/runs/8525288852/job/23351920497?pr=2379

**Comment by dni** [comment]
> @dni can you have a look at this CI failure? Is it related to the PR? https://github.com/lnbits/lnbits/actions/runs/8525288852/job/23351920497?pr=2379

wow, yes, that test saved us :). 61c62f55227d8dcda5173fa1a23563b799df4f49

**Review by prusnak — APPROVED** [review]

**Pull request merged**